libraries/botbuilder/src/teamsActivityHandler.ts

```diff
@@ -9,29 +9,119 @@ import {
     ActivityTypes,
     ActivityHandler,
     AppBasedLinkQuery,
+    ChannelAccount,
+    ChannelInfo,
     FileConsentCardResponse,
     MessagingExtensionAction,
     MessagingExtensionActionResponse,
     MessagingExtensionQuery,
     MessagingExtensionResponse,
     O365ConnectorCardActionQuery,
     SigninStateVerificationQuery,
+    TaskModuleTaskInfo,
     TaskModuleRequest,
     TaskModuleResponse,
+    TeamsChannelData,
+    TeamInfo,
     TurnContext
 } from 'botbuilder-core';
 
 export class TeamsActivityHandler extends ActivityHandler {
 
     /**
      * 
-     * @param handler (context: TurnContext) => Promise<void>
+     * @param context 
      */
-    public onInvokeActivity(handler: (context: TurnContext) => Promise<void>): this {
-        this.checkRegisteredTeamsHandlers('InvokeActivity');
-        return this.on('InvokeActivity', async (context, next) => {
-            await handler(context);
-        });
+    protected async onTurnActivity(context: TurnContext): Promise<void> {
+        if (context.activity.channelId === 'msteams') {
+            // this.teamsRosterClient = 
```

Comment:
>this.teamsRosterClient [](start = 15, length = 22)

Hmm.

Reply:
No impl yet on the class for this member, leaving as is